components/profile_popover/index.js

```diff
@@ -21,6 +29,34 @@ import {getSelectedPost, getRhsState} from 'selectors/rhs';
 
 import ProfilePopover from './profile_popover.jsx';
 
+function hasManageChannelMemberPermission(state) {
```

Comment:
I'm thinking maybe this should be in a `selectors` file instead?

Reply:
Yes, I think so